calc/src/mechanics/util.ts

```diff
@@ -356,6 +357,34 @@ export function getFinalDamage(
   return OF16(pokeRound(Math.max(1, OF32(damageAmount * finalMod) / 0x1000)));
 }
 
+/**
+ * Determines which move category Shell Side Arm should behave as. Applications
+ * that consume this library may want to import this function to show users move
```

Comment:
https://github.com/smogon/damage-calc/blob/master/calc/src/adaptable.ts and https://github.com/smogon/damage-calc/blob/master/calc/src/index.ts is the external API for this package, any other `export`'s are internal. This method is not really of appropriate scope to be included in the top level external API. While you can still depend on internal implementation details, because its not part of the external API there are not going to be guarantees on its existence/behavior.

Reply:
It's definitely going to be useful to me, but I accept this reasoning.  I'll remove that line from the comment.